doc/api/status.rst

```diff
@@ -22,6 +22,10 @@ Or, using the more verbose name:
 Using these constants helps avoid typos and cuts down on the number of
 string objects that must be created when preparing responses.
 
+Falcon also provides a generic 'HTTPStatus' class. Raise this class from a hook,
```

Comment:
What do you think about expanding this to be similar to errors.rst, including pulling in the docstring?